code/src/layouts/app-footer/AppFooter.scss

```diff
@@ -0,0 +1,64 @@
+@import "@/styles/scss/foundation/all";
+
+.footer {
+  padding: spacing(18) 0;
+  background-color: $white;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__list {
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    gap: spacing(2);
+  }
+
+  &__list-title {
+    font-size: $fs-lg;
```

Comment:
I would recommend you to use css var.
For example: (also you can look into AppButton component)
var(spa-list-font-small-size, $fs-lg)

Reply:
Thanks for suggestion, I fixed using similar to `AppButton` approach here and in other places